spec/tests.rb

```diff
@@ -0,0 +1,94 @@
+require_relative '/Users/Sunzi/Desktop/CarCleaningService/Service.rb'
+
+describe 'one car' do
+  subject {service = Service.new}
```

Comment:
`subject { Service.new }` should be enough here. Also I noticed that you are using `subject.add_car` in the tests, should then the subject be `subject { Service.new.add_car }` ?